src/Codeception/Lib/GroupManager.php

```diff
@@ -93,14 +79,58 @@ protected function loadConfiguredGroupSettings()
 
                         $file = trim(Configuration::projectDir() . $test);
```

Comment:
I will delete this line to permit absolute paths in group file.
Relative paths are converted to absolute by normalizeFilePath method below.